src/features/SearchDrug/SearchDrug.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import { ComboBox } from "carbon-components-react";
+import { searchDrugsByName } from "../../utils/CommonUtils";
+import { Title } from "bahmni-carbon-ui";
+
+const SearchDrug = (props) => {
+  const { onChange } = props;
+  const [items, setItems] = useState([]);
+
+  return (
+    <div>
+      <ComboBox
+        id={"SearchDrug"}
+        items={items}
+        placeholder={"Type to Search Drug"}
+        onChange={(e) => {
+          onChange(e.selectedItem);
+        }}
+        size={"xl"}
+        titleText={<Title text={"Drug Name"} isRequired={true} />}
+        onInputChange={async (searchStr) => {
```

Comment:
Move this onchange handler to a separate method and use the reference, avoid using inline,  we can use inline when we have 1 or 2 lines of code in handler. Please check at other files too